functions/postSurveyComplete.protected.ts

```diff
@@ -88,20 +91,37 @@ const saveSurveyInHRM = async (
   });
 };
 
+const getPostSurvetCompleteMessage = (taskLanguage: string | undefined): string => {
+  if (taskLanguage) {
+    try {
+      const translation = JSON.parse(
+        Runtime.getAssets()[`/translations/${taskLanguage}/postSurveyMessages.json`].open(),
+      );
+
+      if (translation.postSurvetCompleteMessage) return translation.postSurvetCompleteMessage;
+    } catch {
+      console.error(`Couldn't retrieve postSurvetCompleteMessage translation for ${taskLanguage}`);
+    }
+  }
+
+  return 'Thank you for reaching out. Please contact us again if you need more help.';
```

Comment:
Should we have the default behaviour use the `en-US` file rather than a hardcoded value if the task language is not set?

If we update the standard message, we might only remember to change it in the translations and forget about the hardcoded fallback

Reply:
My thinking here was that `en-US` might actually be language that's set. I tried to always leave the default message as something hard coded, in case for some reason we can't get the assets, at least the user will receive some message. Happy to change if it's preferred to rely on the jsons exclusively tho!

Reply:
I forgot these translations aren't local to the code. Maybe fall back to en-US and then fall back to a hardcoded string. I'm just aiming for consistency with other translation behaviour